### ISSUE TITLE
html in column names

### ISSUE THREAD

**Issue by r-leyshon** [mwouts/itables#213]
I was attempting to use itables to present units in the column name. I found that the HTML tags work well in cell values but not in table headers:

``` python
from itables import show
test_df = pd.DataFrame(
    {"some_nm<sup>2</sup>": ["some val <sup>2</sup>", "<b>Bold Value</b>"]}
    )
show(test_df)
```

<img width="824" alt="image" src="https://github.com/mwouts/itables/assets/49126943/804801bf-6d81-424f-9afc-2ddc54d091d3">

**Comment by mwouts** [comment]
Hey @r-leyshon , thanks for reporting this! This should be fixed in `itables==1.6.3`, would you mind giving it a try? Thanks

**Comment by r-leyshon** [comment]
@mwouts awesome, and just in time for our presentation. Many thanks!
